sql/analyzer/costed_index_scan.go

```diff
@@ -707,18 +707,18 @@ type indexScanRangeBuilder struct {
 	include   sql.FastIntSet
 	imprecise sql.FastIntSet
 	idToExpr  map[indexScanId]sql.Expression
-	conjIb    *sql.IndexBuilder
-	allRanges sql.RangeCollection
+	conjIb    *sql.MySQLIndexBuilder
```

Comment:
What's the point of this abstraction if we still have mysql-specific types here?